File: samples/ruby/users/me.rb
# retrieve user information
require 'net/http'
require 'json'

API_VERSION = 1
API_HOST    = 'https://api.thebase.in'
CODE        = '' # Your CODE
token       = '' # Retrieve token using ruby/oauth/token.rb

header_parameters =
  { 'Authorization' => "Bearer #{token}" }

uri =
  URI.join(
    API_HOST,
    "#{API_VERSION}/users/me")

request =
  Net::HTTP::Get.new(
    uri,
    header_parameters)

response =
  Net::HTTP.start(
    uri.hostname,
    uri.port,
    use_ssl: uri.scheme == 'https') do |http|
      http.request(request)
    end

parsed_response =
  JSON.parse response.body

File: samples/php/users/me.php
<?php
// ユーザー情報を取得する
define('API_HOST',    'https://api.thebase.in');
define('API_VERSION', '1');
$token = ''; # YOUR ACCESS TOKEN

$headers = array(
    'Authorization: Bearer ' . $token,
);
$request_options = array(
    'http' => array(
        'method' => 'GET',
        'header' => implode("\r\n", $headers),
        'ignore_errors' => true,
    ),
);
$context = stream_context_create($request_options);

$response_body = file_get_contents(API_HOST . '/' . API_VERSION . '/users/me', false, $context);
$parsed_response = json_decode($response_body);
